Documentation/README.md

```diff
@@ -0,0 +1,39 @@
+# Microsoft Azure SDK for .NET Documentation
+
+The following services use AutoRest
+- AppServices
+- Compute
+- DataLake Analytics
+- DataLake Store
+- DataLake Storeuploader
+- DevTestLabs
+- Identity
+- Machine Learning
+- Networking
+- NotificationHubs
+- PowerBI
+- Redis Cache
+- Resource Manager (ARM)
```

Comment:
I think you are missing some. Here is the full list of AutoRest generated and CoreCLR migrated libraries:
```
Microsoft.Azure.Graph.RBAC                             
Microsoft.Azure.Management.Authorization               
Microsoft.Azure.Management.Cdn                         
Microsoft.Azure.Management.CognitiveServices           
Microsoft.Azure.Management.Compute                     
Microsoft.Azure.Management.DataLake.Analytics          
Microsoft.Azure.Management.DataLake.Store              
Microsoft.Azure.Management.DevTestLabs                 
Microsoft.Azure.Management.Dns                         
Microsoft.Azure.Management.MachineLearning             
Microsoft.Azure.Management.Network                     
Microsoft.Azure.Management.NotificationHubs            
Microsoft.Azure.Management.PowerBIEmbedded             
Microsoft.Azure.Management.Redis                    
Microsoft.Azure.Management.ResourceManager            
Microsoft.Azure.Management.Storage                    
Microsoft.Azure.Management.TrafficManager             
Microsoft.Azure.Management.Websites                    
```